src/simparams.jl

```diff
@@ -14,6 +14,8 @@ include("params/strains.jl")
 
 abstract type InfectionModulation end
 
+abstract type InfectionTravels end
```

Comment:
naming: to me travel is one possible execution of imported cases / external cases; therefore the abstract name should refer to that